sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -179,12 +178,41 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
       assert.ok(verifyResult.result);
     });
 
-    it("supports tracing", async function () {
-      await supportsTracing(
-        (tracingOptions) =>
-          cryptoClient.encrypt("RSA1_5", stringToUint8Array("data"), { tracingOptions }),
-        ["Azure.KeyVault.Keys.CryptographyClient.encrypt"]
-      );
+    describe("tracing", () => {
+      it("traces through remote cryptography calls", async () => {
+        if (isLiveMode()) {
+          await assert.supportsTracing(
```

Comment:
So, you don't longer need to import any other code to check if it supports tracing? Is this functionality now part of the test-utils?

Reply:
Yes this functionality is a part of test-utils. There's the supportsTracing chai plugin that you can use to verify tracing.